libsrc/_DEVELOPMENT/target/crt_set_interrupt_mode.inc

```diff
@@ -27,6 +27,17 @@
       ENDIF
       
       ld i,a
+
+      INCLUDE "config_private.inc"
```

Comment:
Setting IL is in the [z180 page zero code](https://github.com/z88dk/z88dk/blob/master/libsrc/_DEVELOPMENT/target/crt_page_zero_z180.inc#L743) but I think you're right that it has to be in the crt if the user is changing the vector table location.